packages/flow-cadut/src/type-checker.js

```diff
@@ -16,7 +16,10 @@
  * limitations under the License.
  */
 
-export const wrongType = type => !type || typeof type != "string"
+export const wrongType = type => !type || typeof type !== "string"
```

Comment:
Maybe `isWrongType` to stay in line with the rest

Reply:
good catch